upgrade/helpers/tasks.py

```diff
@@ -266,23 +320,42 @@ def _sync_sattools_repos_to_satellite_for_capsule(capsuletools_url, org):
             name=customcontents['capsule_tools']['repo'],
             product=captools_product, url=capsuletools_url, organization=org, content_type='yum'
         ).create()
+        logger.info("The custom tools product {} and repository {} is created from "
+                    "capsule tools url".format(customcontents['capsule_tools']['prod'],
+                                               customcontents['capsule_tools']['repo']))
     else:
         captools_product = entities.Product(
             name=rhelcontents['tools']['prod'], organization=org
         ).search(query={'per_page': 100})[0]
         cap_reposet = entities.RepositorySet(
             name=rhelcontents['tools']['repo'].format(sat_ver=to_ver, os_ver=rhelver),
             product=captools_product).search()[0]
+        logger.info("The custom tools product {} and repository {} is created from "
+                    "capsule tools url".format(rhelcontents['tools']['prod'],
+                                               rhelcontents['tools']['repo']))
         try:
             cap_reposet.enable(data={'basearch': arch})
+            logger.info("Capsule repository enabled successfully for arch {} and org {}".
+                        format(arch, org.id))
         except requests.exceptions.HTTPError:
             logger.warn('Check if repository is already enabled, else retrigger')
         time.sleep(5)
         captools_repo = entities.Repository(
             name=rhelcontents['tools']['repofull'].format(
                 sat_ver=to_ver, os_ver=rhelver, arch=arch)
         ).search(query={'organization_id': org.id, 'per_page': 100})[0]
-    call_entity_method_with_timeout(entities.Repository(id=captools_repo.id).sync, timeout=2500)
+        logger.info("Entities repository search completed successfully for tools "
+                    "repo {}".format(rhelcontents['tools']['repofull'].
+                                     format(sat_ver=to_ver, os_ver=rhelver, arch=arch)))
+    logger.info("Entities repository sync started successfully for capsule repo name {}".
+                format(captools_repo.name))
+    start_time = job_execution_time("Entities repository sync")
+    call_entity_method_with_timeout(entities.Repository(id=captools_repo.id).sync, timeout=5000)
```

Comment:
same here ^^

Reply:
To avoid further any timeout-related failure, we analyzed the log after sync.